app/views/survey/_question-answer.html.erb

```diff
@@ -1,5 +1,6 @@
 <li>
-  <input type="radio" name="question-<%= questId %>">
-    <%= answerTitle %>
-  </input>
+
+  <%= radio_button_tag "responses[][answer" + questId.to_s + "]", :answer %>
```

Comment:
I found this in a guide online and I never got a chance to test if it actually works, but the use case seemed right. It's supposed to build the array so that it has the structure of a Responses object. Again, not 100% sure if it actually works. Maybe @seattlecoder9 has some insight?